### PR TITLE
Make /sinks/{sinkId}/edid discoverable and pull up properties under "…

### LINKED ISSUE #17: Should /sinks have similar subroutes to /senders and /receivers?
`/senders/`/ `/receivers/` + `resource ID` return a subroute. `/sinks/` + `Sink ID` returns a Sink JSON, hiding `/edid` subroute. Should we remake this in the consistent way?

### PR DESCRIPTION
**Pull request by N-Nagorny**
…edid" in Sink schema

Resolves #17 and removes `edid` property from Sink schema since EDID may be absent.

### CODE REVIEW

File: examples/sender-get-200.json [hunk 1]
```diff
@@ -1,3 +1,3 @@
 [
-  "media-profiles/"
+  "properties/"
```
Comment (prince-chrismc) [inline examples/sender-get-200.json]: This looks out of place 🤔

**Comment by prince-chrismc** [comment]
Unrelated but I just noticed in the other APIs when there's an "id" in the path the subsequent endpoints have 404 responses documented https://github.com/AMWA-TV/nmos-device-connection-management/blob/ada62dbcc023a0132eddfd9c59cd00ab8a236372/APIs/ConnectionAPI.raml#L199-L200

We are missing those in our RAML

File: examples/sender-get-200.json (hunk 1 above)
Comment (prince-chrismc) [reply examples/sender-get-200.json]:
```suggestion
  "media-profiles/"
```

**Review by prince-chrismc — COMMENTED** [review]
I think you modified one file by accident

**Comment by N-Nagorny** [comment]
@prince-chrismc, thank you for the catch! I committed your fix and added 404.

**Review by prince-chrismc — APPROVED** [review]

File: APIs/SinkMetadataProcessingAPI.raml [hunk 2]
```diff
@@ -117,22 +125,40 @@ mediaType: application/json
...
-      description: Return Sink.
+      description: List all the API endpoints for a single Sink.
       responses:
         200:
           description: Returned when success.
           body:
             example: !include ../examples/sink-get-200.json
-            type: !include schemas/sink.json
-    /edid:
+            type: !include schemas/sink-api-sink.json
```
Comment (garethsb) [inline APIs/SinkMetadataProcessingAPI.raml:134]: FYI, @N-Nagorny, several other specs call these schemas like `base`, e.g. `schemas/sink-base.json` in this case.

**Pull request merged**